### DISCUSSION TITLE
Is there any plan, when 1.27 will be official ?

### DISCUSSION THREAD

**Discussion by DocHifi** [mate-desktop/mate-desktop#598] [General]
Is there any plan, when 1.27 will be official ?
The last Release 1.26 is  from August 2021, now we have 2024.
Or is Mate Desktop dead ?

**Comment by raveit65** [comment]
1.27 stands for development releases.
Next stable major releases will be 1.28.
I took a look in my crystal ball.........and yes i see a plan :-)

**Reply by DocHifi** [reply]
If I look on the Mate Blog, the last is 1.2.6.
https://mate-desktop.org/blog/
So can you tell me what Debian based system use 1.27 ?

**Reply by raveit65** [reply]
I am fedora maintainer and i don't use debian.
I don't think that debian will use development releases.

**Comment by DocHifi** [comment]
> I am fedora maintainer and i don't use debian. I don't think that debian will use development releases.

That was my question, when will 1.2.7  be stable and amiable for all systems.

**Reply by raveit65** [reply]
I did already answered, see my first post.

**Reply by DocHifi** [reply]
You wrote:
Next stable major releases will be 1.28
But not when this will be.
So I see there is no timeline for it.

**Reply by raveit65** [reply]
We don't have a fixed release schedule. It will happen when it happens.
But i see a plan in my crystal ball :-)

**Comment by cweiske** [comment]
1.28 has been released on 2024-02-27: https://mate-desktop.org/blog/2024-02-27-mate-1-28-released/
